### ISSUE TITLE
Creator verified status and "Refresh status" text in Japanese runs onto new lines in Rewards panel

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#19018]
## Description

In Japanese, the text is too long, so it runs onto another line and makes the verified status text not look great.

![image](https://user-images.githubusercontent.com/11497541/138986620-1d8550a0-a874-4cae-9c86-7de271b998f5.png)

cc: @soi-tnk @A-t-s-u-s-h-i

**Comment by Miyayes** [comment]
Even with old translations, I don't see this issue: 
![image](https://user-images.githubusercontent.com/11497541/140101828-8c60cfab-4794-47c6-ac5b-32d8a5845c34.png)

**Comment by A-t-s-u-s-h-i** [comment]
It was fixed on nightly, I think?

This is Brave stable 1.31.88 Chromium: 95.0.4638.69（Official Build） （x86_64）
<img width="390" alt="スクリーンショット 2021-11-04 10 41 19" src="https://user-images.githubusercontent.com/79502386/140243550-b8fca46c-4dad-43c6-b043-687e4b42f4ef.png">


And this is Brave Nightly 1.33.57 Chromium: 95.0.4638.69（Official Build）nightly （x86_64）
<img width="389" alt="スクリーンショット 2021-11-04 7 55 17" src="https://user-images.githubusercontent.com/79502386/140243678-d31d8e6f-f675-449e-b139-a4dcbd7f78a4.png">

Both of page scaling is 100%.

**Comment by soi-tnk** [comment]
Certainly it looks good even with old translation. Should I revert translation from `認証済` to `認証クリエイター` ?

**Comment by Miyayes** [comment]
Okay, it looks fixed in Nightly then. @soi-tnk it's up to you, whichever you think is better Japanese translation for it :). With your new translation, it should continue to look good. 

I'm going to close this for now, since we assume it's fixed in 1.33.x.

**Comment by GeetaSarvadnya** [comment]
Verification passed on

```
Brave	1.33.80 Chromium: 96.0.4664.45 (Official Build) beta （64 ビット）
変更履歴	76e4c1bb2ab4671b8beba3444e61c0f17584b2fc-refs/branch-heads/4664@{#947}
OS	Windows 10 Version 1809 (Build 17763.1637)
```
- Verified the STR from the description

**Verified publisher**
![image](https://user-images.githubusercontent.com/38657976/141976424-46ebd66e-5f70-4c5d-9e7c-e17d9a780581.png)

**Unverified publisher**
![image](https://user-images.githubusercontent.com/38657976/141976830-33fc106b-c73e-4735-a816-699e6c894071.png)
